tests.py

```diff
@@ -0,0 +1,204 @@
+"""This is a test file to test our functions in the Main file."""
+import pytest
+import pytest_cov
+from Bio_stats_assignment import parse_data, num_older_than, sick_patients, patient_age
+
+# test the parse data function
+
+
+def test_parse_data():
+    """Test Parse Data."""
+    # test data with only columns
+    columns_only = "/Users/mohammadanas/Desktop/Duke MIDS/Spring 2021\
+/SoftwareTools/Assignment 4/columns only.txt"
+    data_columns_only = {
+        "PatientID": [],
+        "LabName": [],
+        "LabValue": [],
+        "LabDateTime": [],
+    }
+
+    # test a simple sample data
+    simple_data = "/Users/mohammadanas/Desktop/Duke MIDS/Spring 2021\
+/SoftwareTools/Assignment 4/simple data.txt"
+    data_simple_parse = {
+        "Name": ["Anas", "Jon Snow", "Rumi"],
+        "Age": ["25", "31", "21"],
+        "City": ["Durham", "The Wall", "Ohio"],
+        "Major": ["Data Science", "Ranger", "CS"],
+    }
+
+    # test a patient data
+    patient_test_data = "/Users/mohammadanas/Desktop/Duke MIDS/Spring 2021\
+/SoftwareTools/Assignment 4/Patient test.txt"
+    parsed_patient_test = {
+        "PatientID": ["ANAS123", "JON234", "RUMI874", "JONNA465"],
+        "PatientGender": ["M", "M", "M", "F"],
+        "PatientDateOfBirth": [
+            "1996-04-04 02:45:40.547",
+            "1976-12-28 02:45:40.547",
+            "1984-02-28 02:45:40.547",
+            "1990-07-28 02:45:40.547",
+        ],
+        "PatientRace": ["Asian", "White", "Asian", "White"],
+    }
+
+    # test a sample lab test data
+    labs_data_test = "/Users/mohammadanas/Desktop/Duke MIDS/Spring 2021\
```

Comment:
If you use paths for your machine, no one who clones this repo can run the code. Instead, commit the test data files and make these relative paths.